modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/DDMFormInstanceReportUpgradeProcess.java

```diff
@@ -73,63 +73,58 @@ protected void doUpgrade() throws Exception {
+					"structureVersionId = DDMFormInstanceVersion.",
+					"structureVersionId where DDMFormInstanceRecord.version = ",
+					"DDMFormInstanceRecordVersion.version and ",
+					"DDMFormInstanceRecord.formInstanceId = ? and ",
+					"DDMFormInstanceRecordVersion.status = ?"));
+			PreparedStatement preparedStatement3 =
+				AutoBatchPreparedStatementUtil.concurrentAutoBatch(
+					connection,
+					StringBundler.concat(
+						"insert into DDMFormInstanceReport ",
+						"(formInstanceReportId, groupId, companyId, ",
+						"createDate, modifiedDate, formInstanceId, data_) ",
+						"values (?, ?, ?, ?, ?, ?, ?)"));
+			ResultSet resultSet1 = preparedStatement1.executeQuery()) {
 
 			while (resultSet1.next()) {
 				long formInstanceId = resultSet1.getLong("formInstanceId");
 
 				JSONObject dataJSONObject = _jsonFactory.createJSONObject();
 
-				try (PreparedStatement preparedStatement2 =
-						connection.prepareStatement(
-							StringBundler.concat(
-								"select DDMContent.data_, ",
-								"DDMFormInstanceRecord.formInstanceRecordId, ",
-								"DDMStructureVersion.definition from ",
-								"DDMContent inner join ",
-								"DDMFormInstanceRecordVersion on ",
-								"DDMContent.contentId = ",
-								"DDMFormInstanceRecordVersion.storageId inner ",
-								"join DDMFormInstanceRecord on ",
-								"DDMFormInstanceRecord.formInstanceRecordId = ",
-								"DDMFormInstanceRecordVersion.",
-								"formInstanceRecordId inner join ",
-								"DDMFormInstanceVersion on ",
-								"DDMFormInstanceVersion.formInstanceId = ",
-								"DDMFormInstanceRecordVersion.formInstanceId ",
-								"and DDMFormInstanceVersion.version = ",
-								"DDMFormInstanceRecordVersion.",
-								"formInstanceVersion inner join ",
-								"DDMStructureVersion on ",
-								"DDMStructureVersion.structureVersionId = ",
-								"DDMFormInstanceVersion.structureVersionId ",
-								"where DDMFormInstanceRecord.version = ",
-								"DDMFormInstanceRecordVersion.version and ",
-								"DDMFormInstanceRecord.formInstanceId = ? and ",
-								"DDMFormInstanceRecordVersion.status = ?"))) {
-
-					preparedStatement2.setLong(1, formInstanceId);
-					preparedStatement2.setInt(
-						2, WorkflowConstants.STATUS_APPROVED);
-
-					ResultSet resultSet2 = preparedStatement2.executeQuery();
-
-					while (resultSet2.next()) {
-						dataJSONObject = _processDDMFormValues(
-							dataJSONObject,
-							_getDDMFormValues(
-								resultSet2.getString("data_"),
-								DDMFormDeserializeUtil.deserialize(
-									_ddmFormDeserializer,
-									resultSet2.getString("definition"))),
-							resultSet2.getLong("formInstanceRecordId"));
-
-						dataJSONObject.put(
-							"totalItems",
-							dataJSONObject.getInt("totalItems") + 1);
-					}
+				preparedStatement2.setLong(1, formInstanceId);
+				preparedStatement2.setInt(2, WorkflowConstants.STATUS_APPROVED);
+
+				ResultSet resultSet2 = preparedStatement1.executeQuery();
```

Comment:
@robertoDiaz shouldn't be `ResultSet resultSet2 = preparedStatement2.executeQuery();`?

`preparedStatement2` instead of `preparedStatement1`..

Reply:
you are right. seems to be a typo

Reply:
I'll resend it